build/Makefile

```diff
@@ -122,7 +122,7 @@ ENGINE_DIR = $(F_KORE_DIR)/engine
 
 #  utilities
 SUMMA_NRUTIL= \
-        nrtype.f90 \
+		nrtype.f90 \
```

Comment:
There are a number of whitespace only changes in the makefile.  that is fine but I would suggest standardizing on how you wrap lines.  I like this one but others are less tidy.